controllers/complianceeventsapi/server.go

```diff
@@ -446,9 +503,145 @@ func parseQueryArgs(queryArgs url.Values, isCSV bool) (*queryOptions, error) {
 		}
 	}
 
+	authorizedClusterIds, authorizedClusterNames,
+		unAuthorizedClusters, includeAllManagedClusters, err := getAuthorizedClusters(ctx,
+		db, *parsed, userConfig, client)
+	if err != nil {
+		return nil, err
+	}
+
+	if includeAllManagedClusters {
+		// parsed.Filters["clusters.name"] and parsed.Filters["clusters.cluster_id"] are empty
+		return parsed, nil
+	}
+
+	// If there is any unAuthorized Cluster, it should return ErrNotAuthorized
+	if len(unAuthorizedClusters) > 0 {
+		log.V(2).Info("need GET access", "managedClusters", strings.Join(unAuthorizedClusters, ", "))
```

Comment:
This should be a normal info log and not hidden under the verbosity level of 2 since this is used for audting.